src/test/ui/issue-47184.rs

```diff
@@ -0,0 +1,16 @@
+// Copyright 2015 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+#![feature(nll)]
+
+fn main() {
+    let vec: Vec<&'static String> = vec![&String::new()];
+    //~^ ERROR
```

Comment:
What error do we get now?

Reply:
As far as I can remember (not able to run this at the moment), we got an ICE produced by [this section of the code](https://github.com/davidtwco/rust/blob/e66fa1b3acf4470d0906a413d0b8e0f0abf42a1b/src/librustc_mir/borrow_check/nll/type_check/mod.rs#L768-L775).

Though, strangely, Travis isn't showing this. Perhaps I got mixed up and saw something else.